### ISSUE TITLE
G_loss Label is 1

### ISSUE THREAD

**Issue by liux520** [ptran1203/pytorch-animeGAN#17]
https://github.com/ptran1203/pytorch-animeGAN/blob/a1642dbc6f8119670ff8f57088fbd710cca60019/modeling/losses.py#L111

G_loss label : torch.ones_like(pred)  or  torch.zeros_like(pred) ?

**Comment by ptran1203** [comment]
Maybe a mistake here, I'll try to re-train and confirm this, Thank you
Or you can fire a PR :D
